config/templates/stack-template.json

```diff
@@ -65,7 +64,7 @@
       {{if .Experimental.WaitSignal.Enabled}}
       "CreationPolicy" : {
         "ResourceSignal" : {
-          "Count" : "{{.WorkerCount}}",
+          "Count" : "{{.MinWorkerCount}}",
```

Comment:
In `kube-aws update`, this should match the desired capacity at that time rather than min size, or I guess we will end up destroying the former autoscaling group too early the overall cluster capacity temporarily decreased until  the rest of worker nodes(`DesiredCapacity - MinSize`) becomes ready.

So, do you mind that I'll follow up with a pr addressing this after merging?